.travis.yml

```diff
@@ -6,15 +6,20 @@ cache:
     directories:
         - $HOME/.composer/cache/files
 
-php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - hhvm
+matrix:
+    include:
+        - php: 5.3
+          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
+        - php: 5.4
+        - php: 5.5
+        - php: 5.6
+        - php: 7.0
+          env: DEPENDENCIES=dev
+        - php: hhvm
 
-install: composer install
+before_script:
+    - if [ "$DEPENDENCIES" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi;
+    - composer --prefer-source update $COMPOSER_FLAGS
```

Comment:
please remove the ``--prefer-source``, as it would defeat the usage of the Travis cache to persist the composer cache